packages/instant/src/components/erc20_asset_amount_input.tsx

```diff
@@ -110,10 +122,19 @@ export class ERC20AssetAmountInput extends React.Component<ERC20AssetAmountInput
             currentFontSizePx: fontSizePx,
         });
     };
-    private readonly _handleSymbolClick = () => {
-        if (this.props.onSelectAssetClick) {
-            this.props.onSelectAssetClick(this.props.asset);
+    private readonly _generateSelectAssetClickHandler = (): (() => void) | undefined => {
```

Comment:
perhaps adding a type alias will make line 125 easier to parse:

type ClickHandler = () => void;